connector/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/DockerJDBCIntegrationSuite.scala

```diff
@@ -148,10 +148,6 @@ abstract class DockerJDBCIntegrationSuite
           pulled = true
       }
 
-      docker.pullImageCmd(db.imageName)
-        .start()
-        .awaitCompletion(connectionTimeout.value.toSeconds, TimeUnit.SECONDS)
```

Comment:
Does this really pull the image forcefully? I thought the cache layer is handled by the underlying Docker engine.

Reply:
@dongjoon-hyun, I am not quite sure:)